packages/service-library/src/servicelib/tracing.py

```diff
@@ -40,9 +42,46 @@ def use_tracing_context(context: TracingContext):
             otcontext.detach(token)
 
 
-def setup_log_tracing(tracing_settings: TracingSettings):
-    _ = tracing_settings
-    LoggingInstrumentor().instrument(set_logging_format=False)
+class TracingData(BaseModel):
```

Comment:
I would rename this to `TracingConfig` since `data` implies you are passing in some information about the current trace and can be missleading.